packages/clerk-js/rspack.config.js

```diff
@@ -154,8 +157,8 @@ const svgLoader = () => {
   };
 };
 
-/** @type { () => (import('@rspack/core').RuleSetRule[]) } */
-const typescriptLoaderProd = () => {
+/** @type { (opts: { targets: string, useCoreJs?: boolean }) => (import('@rspack/core').RuleSetRule[]) } */
+const typescriptLoaderProd = ({ targets, useCoreJs = false }) => {
```

Comment:
Nit:

What about using `packageJSON.browserslist` as a default value for `targets`? We probably (with our current knowledge/approach) want to change the targets for more variants, so that could clean up some repetition

Reply:
ooo great idea!